demo/src/components/View3D.tsx

```diff
@@ -11,7 +12,7 @@ interface DemoOptions extends Partial<View3DOptions> {
   className: string;
   clickToLoad: boolean;
   showARButton: boolean;
-  showLoadingBar: boolean | string;
+  showLoadingBar: LoadingBarOptions | null;
```

Comment:
Wouldn't it be dangerous to change showLoadingBar from boolean to object?

Reply:
It seems that boolean values should also be included.

Reply:
That's the component only for our demo page, so that's fine.